spec/features/admin_spec.rb

```diff
@@ -0,0 +1,43 @@
+feature "admin" do
+  user = FactoryGirl.create(:user, admin: true)
+  other_user = FactoryGirl.create(:user)
+  movie = FactoryGirl.create(:movie)
+  
+  before(:each) do
+    visit(root_path)
+    click_link 'Sign In'
+    fill_in("Email", with: user.email)
+    fill_in("Password", with: user.password)
+    click_button("Log in")
+  end
+
+  scenario "views a list of users" do
+    click_link "All Users"
+    expect(page).to have_content("Users List")
+    expect(page).to have_content(other_user.username)
+    expect(page).to have_content(user.username)
+  end
+
+  scenario "can delete a user" do
+    click_link "All Users"
+    click_button "Delete #{other_user.username}"
+    expect(page).to_not have_content(other_user.username)
+  end
```

Comment:
You should have an expect statement for testing the flash message that says "User Deleted" upon a successful request.